src/Avalonia.Controls/VirtualizingStackPanel.cs

```diff
@@ -51,7 +52,25 @@ public class VirtualizingStackPanel : VirtualizingPanel, IScrollSnapPointsInfo
             RoutedEvent.Register<VirtualizingStackPanel, RoutedEventArgs>(
                 nameof(VerticalSnapPointsChanged),
                 RoutingStrategies.Bubble);
+        /// <summary>
+        /// Defines the <see cref="BufferFactor"/> property.
+        /// </summary>
+        public static readonly StyledProperty<double> BufferFactorProperty =
+            AvaloniaProperty.Register<VirtualizingStackPanel, double>(nameof(BufferFactor), 0.0, 
+                validate: v => v is >= 0 and <= 2);
 
+        /// <summary>
+        /// Gets or sets the factor to determine how much additional space to maintain above and below the viewport.
```

Comment:
Nit: `<summary>` sections should be a single sentence with the remainder going into `<remarks>` (i can't find the docs for this right now, but if you look at the MS API docs you'll see that is generally the guideline)